OmiseSDKTests/PaymentInformationTestCase.swift

```diff
@@ -647,12 +647,16 @@ class PaymentInformationTestCase: XCTestCase {
         let encoder = PaymentInformationTestCase.makeJSONEncoder()
 
         do {
+            let shipping = PaymentInformation.Atome.ShippingAddress(
```

Comment:
@vault087 Do you mean to have Atome changes as part of this PR?

Reply:
Unfortunately this change accidentally come to this PR as this tasks was part of Atome. It doesn't affect Atome anyhow though so I just leave it here, it's already fixed in latest related PRs